src/main/java/pages/TicketPricePage.java

```diff
@@ -1,19 +1,35 @@
 package pages;
 
 
-import base.WebDriverConfig;
+import base.DriverManager;
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import utils.SeleniumHelper;
 
 public class TicketPricePage {
 
-    private String seat = "//*[@id='content']/table/tbody/tr[td[text()='%s']]//a[@class='BoxLink'][normalize-space()='Book ticket']";
-
+    private final String seatXPath = "//*[@id='content']/table/tbody/tr[td[text()='%s']]//a[@class='BoxLink'][normalize-space()='Book ticket']";
+    private final String titleTicketPriceXPath = "//th[contains(text(),'Ticket price from %s to %s')]";
+    private final String priceSeatTypeXPath = "//table[@class='MyTable MedTable']//th[normalize-space()='Price (VND)']//following-sibling::td[count(//td[text()='%s']/preceding-sibling::td)+1]";
 
     public void chooseTypeSeat(String typeSeat) {
-        String xpathExpression = String.format(seat, typeSeat);
-        WebElement chooseSeat = WebDriverConfig.driver.findElement(By.xpath(xpathExpression));
-        chooseSeat.click();
+        String xpathExpression = String.format(seatXPath, typeSeat);
+        By chooseSeat = By.xpath(xpathExpression);
+        SeleniumHelper.clickElement(chooseSeat);
+    }
+
+    public boolean verifyTitleTable(String departStation, String arriveAt) {
```

Comment:
rename method, for these methods return boolean, it's better if the prefix is 'is/does...' (isErrorMessageDisplayed, doesControlExist)